app/serializers/dist/json_serializer.rb

```diff
@@ -70,5 +72,61 @@ def serialize_nested(connection)
         name: connection.name,
       }
     end
+
+    def build_mapping_blocks(mapping_rules)
```

Comment:
Use the instance variable instead of passing this in? The other ones take a param because they're operating against a subset of the instance var vs its entirety